spec/index.bs

```diff
@@ -2786,6 +2786,32 @@ interface GPUCommandEncoder {
 GPUCommandEncoder includes GPUObjectBase;
 </script>
 
+{{GPUCommandEncoder}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPUCommandEncoder">
+    : <dfn>\[[state]]</dfn> of type [=encoder state=].
+    ::
+        The current state of the {{GPUCommandEncoder}}, initially set to [=encoder state/open=].
+</dl>
+
+Each {{GPUCommandEncoder}} has a current <dfn>encoder state</dfn> on the [=Content timeline=]
+which may be one of the following:
+
+ - "<dfn for="encoder state">open</dfn>" where the {{GPUCommandEncoder}} is available to begin new operations.
+ - "<dfn for="encoder state">recording a render pass</dfn>" where the {{GPUCommandEncoder}} has an active {{GPURenderPassEncoder}}.
+ - "<dfn for="encoder state">recording a compute pass</dfn>" where the {{GPUCommandEncoder}} has an active {{GPUComputePassEncoder}}.
+ - "<dfn for="encoder state">closed</dfn>" where the {{GPUCommandEncoder}} is
+     no longer available for any operations.
+
+<div class=note>
+    {{GPUCommandEncoder}} has a state machine where the states are:
+
+     - [=encoder state/open=] when initially created until {{GPUComputePassEncoder/finish()}} is called and when the {{GPUCommandEncoder}} has no active {{GPURenderPassEncoder}} or {{GPUComputePassEncoder}}.
```

Comment:
nit: please split in multiple lines for ease of review in GH

Reply:
Sure! Is there a recommended max line width? There doesn't seem to be much consistency in the spec as it stands today.

Reply:
yeah, it's rather arbitrary. You can see for yourself how it looks in the review:) Keeping it mostly between 80 and 100 worked well so far, but it's not a strict requirement.